src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx

```diff
@@ -23,9 +23,11 @@ function TextProposalForm() {
   const handleCancelClick = () => navigate("/governance");
 
   async function onSubmit(data: TextProposalFormData) {
-    // TODO: Proposal Contract functions do not handle both title and description yet.
-    // description: data.description,
-    await governance.createProposal(ProposalType.Text, { title: data.title });
+    await governance.createProposal(ProposalType.Text, {
+      title: data.title,
+      description: data.description,
+      linkToDiscussion: "",
```

Comment:
We'll need to add the discussion link input component to the Text Proposal form in a follow up PR.

Reply:
@damiano-melcarne-lab49  I have addressed this under https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/pull/88, one this PR is merged I can merge mine.

cc: @shivamgarglab49